index.bs

```diff
@@ -309,7 +309,13 @@ Every {{Document}} has a <dfn for="Document">post-prerendering activation steps
 
       1. Set |successorBC|'s [=browsing context/loading mode=] to "`default`".
 
-      1. [=Fire an event=] named {{Document/prerenderingchange}} at |successorBC|'s [=active document=].
+      1. Let |d| be |successorBC|'s [=active document=].
+
+      1. [=Fire an event=] named {{Document/prerenderingchange}} at |d|.
+
+      1. [=list/For each=] |descendantBC| of |d|'s <a spec="HTML">list of the descendant browsing contexts</a>, [=queue a global task=] on the [=networking task source=], given |descendantBC|'s [=browsing context/active window=], with steps:
+
```

Comment:
When do we run descendantBC's active document's post-prerendering activation steps list?

Reply:
We currently do not. Per the spec, we only add to these steps for a prerendering browsing context which excludes subframes. It's a good point that's probably not the right behavior. IIUC, we currently use this to defer a download until activation - but I don't think this covers the case where a subframe in the prerender initiates a download.